how-to/customize-workspace/client/src/framework/bootstrapper.ts

```diff
@@ -42,7 +42,7 @@ const logger = createLogger("Bootstrapper");
 
 let bootstrapOptions: BootstrapOptions;
 
-export async function init() {
+export async function init(): Promise<boolean> {
```

Comment:
Any reason this now always returns `true`, why not void ?

Reply:
This was in prep for if we wanted to be able to return a chain of bools (e.g. bootstrapper true, return that value up from provider so consumers know that everything works. At the moment nothing is driving the true so it could be a void but was just planning ahead.